mozci/data/sources/activedata/__init__.py

```diff
@@ -58,6 +58,17 @@ def normalize(cls, result):
                 # We don't need to store the retry ID.
                 del item["retry_id"]
 
+            # Normalize result
+            result_map = {
+                "success": "passed",
+                "testfailed": "failed",
+                "bustage": "failed",
+                "usercancel": "canceled",
+                "retry": "exception",
+            }
```

Comment:
`superseded` is not supported by AD?

Reply:
It is, it's just named the same so doesn't need translation.